### PR TITLE
chore: Mass Integration 0.19

### PR DESCRIPTION
**Pull request by dcampora**
# Mass integration 0.19

## GitHub Bot Help

`/bot [-h] ['run', 'kill', 'skip', 'reuse-pipeline'] ...`

Provide a user friendly way for developers to interact with a Jenkins server.

Run `/bot [-h|--help]` to print this help message.

See details below for each supported subcommand.

<details>

`run  [--disable-fail-fast --skip-test --stage-list "A10-1, xxx" --gpu-type "A30, H100_PCIe" --add-multi-gpu-test --only-multi-gpu-test --disable-multi-gpu-test --post-merge --extra-stage "H100_PCIe-[Post-Merge]-1, xxx"]`

Launch build/test pipelines. All previously running jobs will be killed.

`--disable-fail-fast ` *(OPTIONAL)* : Disable fail fast on build/tests/infra failures.

`--skip-test ` *(OPTIONAL)* : Skip all test stages, but still run build stages, package stages and sanity check stages. Note: Does **NOT** update GitHub check status.

`--stage-list "A10-1, xxx"` *(OPTIONAL)* : Only run the specified test stages. Examples: "A10-1, xxx". Note: Does **NOT** update GitHub check status.

`--gpu-type "A30, H100_PCIe"` *(OPTIONAL)* : Only run the test stages on the specified GPU types. Examples: "A30, H100_PCIe". Note: Does **NOT** update GitHub check status.

`--only-multi-gpu-test ` *(OPTIONAL)* : Only run the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--disable-multi-gpu-test ` *(OPTIONAL)* : Disable the multi-GPU tests. Note: Does **NOT** update GitHub check status.

`--add-multi-gpu-test ` *(OPTIONAL)* : Force run the multi-GPU tests. Will also run L0 pre-merge pipeline.

`--post-merge ` *(OPTIONAL)* : Run the L0 post-merge pipeline instead of the ordinary L0 pre-merge pipeline.

`--extra-stage "H100_PCIe-[Post-Merge]-1, xxx"` *(OPTIONAL)* : Run the ordinary L0 pre-merge pipeline and specified test stages. Examples: --extra-stage "H100_PCIe-[Post-Merge]-1, xxx".

### kill

`kill  `

Kill all running builds associated with pull request.

### skip

`skip --comment COMMENT `

Skip testing for latest commit on pull request. `--comment "Reason for skipping build/test"` is required. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

### reuse-pipeline

`reuse-pipeline `

Reuse a previous pipeline to validate current commit. This action will also kill all currently running builds associated with the pull request. IMPORTANT NOTE: This is dangerous since lack of user care and validation can cause top of tree to break.

</details>

### CODE REVIEW

**Review by litaotju — APPROVED** [review]

File: tensorrt_llm/llmapi/mgmn_worker_node.py [hunk 1]
```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env python3
 import logging
 
+import torch
```
Comment (byshiue) [inline tensorrt_llm/llmapi/mgmn_worker_node.py:4]: @dcampora We don't require the changes in this file. It is a workaround on 0.19 and we have officail fixes in main branch. So, you can revert all changes in this file.

**Review by nv-guomingz — APPROVED** [review]
LGTM

**Review by kaiyux — APPROVED** [review]

**Review by hyukn — APPROVED** [review]

**Review by amukkara — APPROVED** [review]

File: tensorrt_llm/llmapi/mgmn_worker_node.py (hunk 1 above)
Comment (dcampora) [reply tensorrt_llm/llmapi/mgmn_worker_node.py:4]: Thanks, done.

**Review by Superjomn — APPROVED** [review]
LGTM

**Review by jiahanc — APPROVED** [review]

File: tests/integration/defs/accuracy/test_llm_api_pytorch.py [hunk 2]
```diff
@@ -436,6 +436,24 @@ def test_fp8_block_scales(self, mtp_nextn, fp8kv, attention_dp, cuda_graph,
             task = GSM8K(self.MODEL_NAME)
             task.evaluate(llm)
 
+    @pytest.mark.skip_device_not_contain(["H100"])
+    def test_fp8_block_scales_cuda_graph_padding(self):
+        # OOM on H100 with default free_gpu_memory_fraction=0.9
+        kv_cache_config = KvCacheConfig(free_gpu_memory_fraction=0.8)
+        pytorch_config = PyTorchConfig(enable_overlap_scheduler=True,
```
Comment (liji-nv) [inline tests/integration/defs/accuracy/test_llm_api_pytorch.py:443]: Change is outdated. The flag now is disable_overlap_scheduler

**Review by liji-nv — CHANGES_REQUESTED** [review]

File: tests/integration/defs/accuracy/test_llm_api_pytorch.py (hunk 2 above)
Comment (dcampora) [reply tests/integration/defs/accuracy/test_llm_api_pytorch.py:443]: Changed.

**Review by liji-nv — APPROVED** [review]

File: tests/integration/defs/accuracy/test_llm_api_pytorch.py (hunk 2 above)
Comment (liji-nv) [reply tests/integration/defs/accuracy/test_llm_api_pytorch.py:443]: I still can see the enable_overlap_scheduler arg.

Comment (dcampora) [reply tests/integration/defs/accuracy/test_llm_api_pytorch.py:443]: Where?

Comment (dcampora) [reply tests/integration/defs/accuracy/test_llm_api_pytorch.py:443]: My bad, my git push command failed. Looking into it.

Comment (dcampora) [reply tests/integration/defs/accuracy/test_llm_api_pytorch.py]: Now my git push force command worked.
